script/Ciel/Day 9/09_01_CIEL09_1_2.txt

```diff
@@ -6,18 +6,17 @@
 [sha:7c615c45aee80a4a59ffac852decc1fb96bd6239]{
 -- Page 10004, Offset 22335.
 -- 『良薬は口に苦し』の逆をいく注射がどんなものか、興味が湧いた。
-I'm curious to see how it's going to feel, an injection that's going against the principle of 'A good medicine is one that tastes bitter'.
+It's an injection that goes against the principle of 'good medicine tastes bitter'. My curiosity is piqued.
 }
 [sha:a9878d320a0e3a0797ad5520b89c57e08b4bfebb]{
 -- Page 79, Offset 22336.
 -- 「え、ホント？　こっち、使っていいの？　こんなノンケの美少年に？
-// 'Who doesn't take it that way' maybe? Unless not clear enough
-"Huh, really? You want me to use this one? You, a young dashing man who isn't swinging that way?
+"Huh, really? You want me to use this one? On you, a young dashing man who doesn't swing that way?
 }
 [sha:77f7b48e3a0c4b40788f28f9530bf9df57842882]{
 -- Page 79, Offset 22337. Mods: @n.
 -- うわやば、アタシもってる。間違いなく蠍座の女の<星辰|うんせい>きちゃってる。
-Oh gawds, I'm in luck. I am blessed by my fortune as a Scorpio woman!
+Oh gawds, I'm in luck. This must be my fortune as a Scorpio woman!
```

Comment:
"Oh gawds"??? Just... no. "My god, I'm in luck" or "I'm so lucky" or something. Anything but "oh gawds"